.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+*                                    @MetaMask/devs
```

Comment:
Might be worth including more people here :thinking: This repo is somewhat unique in that we have many people helping to maintain it.

Reply:
Makes sense! Anyone in particular?

Reply:
Probably best to raise this with the wider team on Monday. I'm not sure that I'm up-to-date on how this repo is currently being managed.

Reply:
added @xrn

Reply:
@409H - would you like to be added as a code owner as well?

Reply:
@ryanml Sure, I'll accept!

Reply:
I've added you, thanks!